src/app/loadout-builder/generated-sets/GeneratedSetButtons.tsx

```diff
@@ -47,8 +49,13 @@ export default function GeneratedSetButtons({
           {t('LoadoutBuilder.CompareLoadout')}
         </button>
       )}
-      <button type="button" className="dim-button" onClick={openLoadout}>
-        {t('LoadoutBuilder.CreateLoadout')}
+      <button
```

Comment:
What is your thoughts on disabling this button when a loadout exists? Part of me thinks it should be disabled because it's no longer a create loadout button but maybe people will want to create a new one with those items anyway? 

The idea of still being able to create a new loadout makes me wonder whether it should be some other indicator on the loadout rather than replacing the button. 

I am happy to go with your instinct on this but thought it was worth bringing up.

Reply:
I don't want to be the one to remove any functionality - some people may want to knowingly duplicate loadouts this way.

There's this option?
<img width="730" alt="loadout-exists-" src="https://user-images.githubusercontent.com/58656026/93709045-0a201500-fb33-11ea-91d5-adabd3594615.png">

Row height calculation needs some work, but other than that I think I prefer it to the button option.

Reply:
I would probably cap the width and use text-overflow to create ellipsis. Then it could line up with the first button. 

In saying that after playing around with some sizing on this page that area under the buttons seems to be wasted space. We should probably rework the css so we can utilise it as we add more functionality like this. Then we wouldn't need to increase all the row heights (if you change one you change them all).

It could be done in another PR though so don't feel you have to do it.

Reply:
Having the message under the buttons either creates issues with rowHeight or with mobile layout. I've moved the message underneath the stat bar and increased the header height by ~11px for each row to accommodate it, seems to work out without massively inflating the look of the other rows.

<img width="730" alt="chrome_l7EofMAo3K" src="https://user-images.githubusercontent.com/58656026/93762518-a61b5080-fc07-11ea-971c-43b40cce3167.png">
<img width="304" alt="chrome_FaOOLBKPYK" src="https://user-images.githubusercontent.com/58656026/93762528-a9164100-fc07-11ea-9864-f34e7f093560.png">

I would like to take a run at the layout in a future PR though, especially for mobile. Lots of dead space at the moment.

Reply:
Yeah that would be excellent